### PR TITLE
define database table

### PR DESCRIPTION
**Pull request by bellatoris**
rank에 대한 디비 쿼리와 모델을 정의합니다.

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #140** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/140" target="_blank">Graphite</a> 
    * **PR #141** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/141" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/141?utm_source=stack-comment).

File: subprojects/bounded_context/accessibility/domain/src/main/kotlin/club/staircrusher/accessibility/domain/model/AccessibilityRank.kt [hunk 1]
```diff
@@ -1,8 +1,12 @@
 package club.staircrusher.accessibility.domain.model
 
+import java.time.Instant
+
 data class AccessibilityRank(
     val id: String,
     val userId: String,
     val conquestCount: Int,
-    val rank: Long,
+    val rank: Long?,
```
Comment (Zeniuus) [inline subprojects/bounded_context/accessibility/domain/src/main/kotlin/club/staircrusher/accessibility/domain/model/AccessibilityRank.kt:9]: 왜 nullable이 됐어요

**Review by Zeniuus — APPROVED** [review]

File: subprojects/bounded_context/accessibility/domain/src/main/kotlin/club/staircrusher/accessibility/domain/model/AccessibilityRank.kt (hunk 1 above)
Comment (bellatoris) [reply subprojects/bounded_context/accessibility/domain/src/main/kotlin/club/staircrusher/accessibility/domain/model/AccessibilityRank.kt:9]: 우선 rank없이 conquest count만 입력한 후에 rank를 업데이트 할 생각이었습니다 ㅋㅋ

Comment (Zeniuus) [reply subprojects/bounded_context/accessibility/domain/src/main/kotlin/club/staircrusher/accessibility/domain/model/AccessibilityRank.kt:9]: 후속 PR에서 확인 완료

**Comment by bellatoris** [comment]
@bellatoris queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/141).

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/141) rebased this pull request as part of a merge.

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/141).

**Pull request merged**